### PR TITLE
CollectDependencies(): Fix a performance issue

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: model/src/main/kotlin/Scope.kt [hunk 1]
```diff
@@ -51,12 +51,15 @@ data class Scope(
      * depth. If [includeErroneous] is true, [PackageReference]s with issues (but not their dependencies without issues)
```
Comment (tsteenbe) [inline model/src/main/kotlin/Scope.kt]: Comment statement needs to be updates as `includeErroneous` has been renamed to `filterPredicate`

Comment (fviernau) [reply model/src/main/kotlin/Scope.kt]: Done.

**Comment by fviernau** [comment]
@tsteenbe thanks for the comments.
I just updated the PR, hope all your comments are reasonably addressed.

**Comment by fviernau** [comment]
Merge regardlessly of appveyor issues as these are not related.

**Pull request merged**